crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -624,6 +582,22 @@ impl<'w, 's> SystemParamFetch<'w, 's> for WorldState {
 /// // Note how the read local is still 0 due to the locals not being shared.
 /// assert_eq!(read_system.run((), world), 0);
 /// ```
+///
+/// N.B. A [`Local`]s value cannot be read or written to outside of the containing system.
+/// To add configuration to a system, convert a capturing closure into the system instead:
+///
+/// ```
+/// # use bevy_ecs::prelude::*;
+/// # use bevy_ecs::system::assert_is_system;
+/// struct Config(u32);
+/// struct Myu32Wrapper(u32);
+/// fn reset_to_system(value: Config) -> impl FnMut(ResMut<Myu32Wrapper>) {
+///     move |mut val| val.0 = value.0
+/// }
+///
+/// // .add_system(reset_to(my_config))
```

Comment:
could this example be more complete? you missed a rename here as it's commented

Reply:
We're stuck in bevy_ecs here, so I can't create an `App`, without making a stub implementation. That seemed like quite a lot of effort for the example.

There's not really a great way to demonstrate adding systems within `bevy_ecs` imo, since every consumer should also be using `bevy_app`.

The rename does need to happen though, thanks.